Yona/Yona/UserRequestManager.swift

```diff
@@ -75,7 +75,14 @@ class UserRequestManager{
     
     
     func postOpenAppEvent(_ user: Users, onCompletion: @escaping APIResponse) {
-        if let path = user.openAppEventLink, let appVersion = Bundle.main.infoDictionary?["CFBundleShortVersionString"]  as? String, let appVersionCode = Bundle.main.infoDictionary?["CFBundleVersion"]  as? String {
+        var openAppEventLinkPath: String?
```

Comment:
This, along with the below for-loop should be reducible to something like this:
```
let openAppEventLinkPath = user.devices.first(where: { $0.isRequestingDevice }).map {$0.openAppEventLink}
```

Reply:
Done.